src/utils/update/managing.officer.mapper.ts

```diff
@@ -0,0 +1,77 @@
+import { mapAddress, isSameAddress } from "./mapper.utils";
+import { CompanyOfficer } from "@companieshouse/api-sdk-node/dist/services/company-officers/types";
+import { ManagingOfficerIndividual } from "../../model/managing.officer.model";
+import { ManagingOfficerCorporate } from "../../model/managing.officer.corporate.model";
+import { InputDate, yesNoResponse } from "../../model/data.types.model";
+
+export const mapToManagingOfficer = (officer: CompanyOfficer): ManagingOfficerIndividual => {
+  console.log(officer);
+  const raw = officer as any;
+  const service_address = mapAddress(officer.address);
+  const address = mapAddress(officer.address);
+  const names = splitNames(officer.name);
+
+  return {
+    id: raw.links?.self,
+    links: {
+      self: raw.links?.self
+    },
+    first_name: names[0],
+    last_name: names[1],
+    has_former_names: raw.formerNames ? yesNoResponse.Yes : yesNoResponse.No,
+    former_names: undefined,
```

Comment:
This will need to come from raw.formerNames

Reply:
Needs to be converted to string. All the names with " " between forenames & surname and "," separation between former names.